index.bs

```diff
@@ -934,33 +958,9 @@ enum WebTransportState {
 :: The transport has completed negotiation of a secure connection. Outgoing
    data and media can now flow through.
 : <dfn for="WebTransportState" enum-value>"closed"</dfn>
-:: The transport has been closed intentionally via a call to
-   {{WebTransport/close()}} or receipt of a closing message from the remote
-   side.  When the {{WebTransport}}'s [=[[State]]=]
-   transitions to `closed` the user agent MUST run the following steps:
-
-   1. Let |transport| be the {{WebTransport}}.
-   1. Close the {{ReadableStream}} in |transport|'s [=[[ReceivedStreams]]=].
-   1. Close the {{ReadableStream}} in |transport|'s
-      [=[[ReceivedBidirectionalStreams]]=].
```

Comment:
Good riddance. Just noting a discrepancy here that this old logic suggested we "close"  the stream-of-stream objects, whereas the new [cleanup](https://w3c.github.io/webtransport/#webtransport-cleanup) procedure errors them. I think the latter is correct, since merely closing those streams wouldn't be abrupt enough (they couldn't keep function in this state). Just wanted to make sure this was intentional.